content/chapters/compute/lab/content/synchronisation.md

```diff
@@ -128,7 +128,7 @@ We read the counter, increment (modify) it and then write it back.
 Think about our example with [`apache2`](./processes-threads-apache2.md)
 Let's say a `worker` has created a _pool_ of 3 threads.
 They are not doing any work initially;
-they are in the WAITING state.
+they are in the **WAITING** state.
```

Comment:
Not needed. I prefer to only use bold when introducing a term/concept. By this point, the WAITING state should be known by students.